From 80fc4bf72bc252b9a0469f9fb7d8523edfa57cfa Mon Sep 17 00:00:00 2001
From: Richard <peper03@yahoo.com>
Date: Sat, 18 May 2013 01:09:06 +0200
Subject: [PATCH 236/285] Added missing initialisation of MHEG member variable
 that sometimes caused (non-MHEG) video playback to
 pause at the end.

Signed-off-by: Stuart Morgan <smorgan@mythtv.org>
Signed-off-by: Lawrence Rust <lvr@softsystem.co.uk>
---
 mythtv/libs/libmythtv/mhi.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mythtv/libs/libmythtv/mhi.cpp b/mythtv/libs/libmythtv/mhi.cpp
index a60d599..ed0ef84 100644
--- a/mythtv/libs/libmythtv/mhi.cpp
+++ b/mythtv/libs/libmythtv/mhi.cpp
@@ -53,7 +53,7 @@ class MHIImageData
 static pthread_t kNullThread;
 MHIContext::MHIContext(InteractiveTV *parent)
     : m_parent(parent),     m_dsmcc(new Dsmcc()),
-      m_keyProfile(0),
+      m_notify(0),          m_keyProfile(0),
       m_engine(MHCreateEngine(this)), m_stop(false),
       m_stopped(true),      m_updated(false),
       m_face_loaded(false), m_engineThread(kNullThread),
-- 
1.7.9.5

